### PR TITLE
Remove Config

### PR DESCRIPTION
**Pull request by robehn**
For you to consider.
- Adds 1-2ns (thaw+freeze) (<0.5%)
- Passes t1-3 (no new issues found)
- Reduces compilation time with over 30 seconds (15-20%).
- Loom have the same compile time as vanilla jdk/jdk.

---------
### Progress
- [x] Change must not contain extraneous whitespace

### Reviewers
 * [Coleen Phillimore](https://openjdk.java.net/census#coleenp) (@coleenp - Committer) ⚠️ Review applies to 858efba85e544fcdef487b84e64be556a4e5f886
 * [Erik Österlund](https://openjdk.java.net/census#eosterlund) (@fisk - Committer) ⚠️ Review applies to 07af4de3d7e7c152f46b7e953456cda82d30c067

### Reviewing
<details><summary>Using <code>git</code></summary>

Checkout this PR locally: \
`$ git fetch https://git.openjdk.java.net/loom pull/110/head:pull/110` \
`$ git checkout pull/110`

Update a local copy of the PR: \
`$ git checkout pull/110` \
`$ git pull https://git.openjdk.java.net/loom pull/110/head`

</details>
<details><summary>Using Skara CLI tools</summary>

Checkout this PR locally: \
`$ git pr checkout 110`

View PR using the GUI difftool: \
`$ git pr show -t 110`

</details>
<details><summary>Using diff file</summary>

Download this PR as a diff file: \
<a href="https://git.openjdk.java.net/loom/pull/110.diff">https://git.openjdk.java.net/loom/pull/110.diff</a>

</details>

### CODE REVIEW

**Comment by pron** [comment]
We'll get to this a bit later :)

File: src/hotspot/share/runtime/continuation.cpp [hunk 1]
```diff
@@ -2609,8 +2569,8 @@ class Thaw : public ThawBase {
     // we need to clear the bits that correspond to arguments as they reside in the caller frame
     log_develop_trace(jvmcont)("clearing bitmap for " INTPTR_FORMAT " - " INTPTR_FORMAT, p2i(start), p2i(start+range));
     stackChunkOop chunk = _cont.tail();
-    chunk->bitmap().clear_range(chunk->bit_index_for((typename ConfigT::OopT*)start),
-                                chunk->bit_index_for((typename ConfigT::OopT*)(start+range)));
```
Comment (coleenp) [inline src/hotspot/share/runtime/continuation.cpp:2613]: I hate C++ template syntax.

**Review by coleenp — APPROVED** [review]
Looks good.  This helps readability and we need convince ourselves that there's no other way to get this small bit of performance back other than these templates.

**Review by fisk — APPROVED** [review]
Looks good.

**Comment by coleenp** [comment]
more whitespace errors...

**Comment by coleenp** [comment]
Are we ready for this one yet?

**Comment by AlanBateman** [comment]
Should this be closed?

**Pull request closed without merging**